### DISCUSSION TITLE
Jewelcrafting Trainer in capitals

### DISCUSSION THREAD

**Discussion by MrWallace78** [chromiecraft/chromiecraft#7293] [Suggestions]
Now Jelcrafting Trainers are only in Exodar and Silvermoon city? Is it possible to have them in the other capitals?

**Comment by cachocarne** [comment]
It's not "now". This is how it was in TBC and during WotLK, which is the version of the game we are playing. Both cities are actually on different maps than the rest of Azeroth. You will also find plenty of trainers in Outland.
